apps/precinct-scanner/src/AppRoot.tsx

```diff
@@ -599,6 +611,9 @@ const AppRoot = ({
 
         // Figure out what ballot state we are in, defaulting to the current state.
         switch (scannerState) {
+          case ScannerStatus.Crashed:
```

Comment:
This will make the new screen show up if the crashed status is returned NOT while a scan is in progress, not sure if we actually want that.

Reply:
is it ONLY when there is no scan in progress, or both when and not when?